src/test/java/uk/gov/hmcts/reform/userprofileapi/domain/enums/IdentifierNameTest.java

```diff
@@ -0,0 +1,22 @@
+package uk.gov.hmcts.reform.userprofileapi.domain.enums;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.junit.Test;
+
+public class IdentifierNameTest {
+
+    @Test
+    public void should_return_correct_string_values() {
+        assertThat(IdentifierName.EMAIL.toString()).isEqualTo("EMAIL");
+        assertThat(IdentifierName.UUID.toString()).isEqualTo("UUID");
+        assertThat(IdentifierName.UUID_LIST.toString()).isEqualTo("UUID_LIST");
+    }
+
+    @Test
+    public void should_return_correct_enum_from_string() {
+        assertThat(IdentifierName.valueOf("EMAIL")).isEqualTo(IdentifierName.EMAIL);
+        assertThat(IdentifierName.valueOf("UUID")).isEqualTo(IdentifierName.UUID);
+        assertThat(IdentifierName.valueOf("UUID_LIST")).isEqualTo(IdentifierName.UUID_LIST);
+    }
```

Comment:
these tests dont look right.. we need to find a better way to do the unit tests for enums.
I have spoken to Shreedhar and given couple of options...we need to try and see which will give coverage

Reply:
Sure, I simply followed how the other Enum tests had been implemented before

Reply:
We could always add this to the ticket regarding fixing how the exception tests are carried out